config/create-settings.js

```diff
@@ -11,7 +11,8 @@ function createSettings(options) {
       BASE_URL: process.env.BASE_URL
     },
     vic: {
-      rateLimit: options.buildtype === 'development' ? 1 : 0.1
+      rateLimitAuthed: 1,
```

Comment:
@jkassemi Just realized: if we default this to 1, will we have to tweak it on S3 after every deploy? Until we get to the point where we're not rate-limiting, that is

Reply:
Yeah, with this implementation that is correct.

Reply:
OK, then we should probably change this default to 0.05 for now. We'll still have to tweak on deploys since it's looking like we'll want to ramp up and down throughout the week.